### PR TITLE
validate also after value change

### PR DESCRIPTION
**Pull request by MJedr**
ref:inspirehep/inspirehep#2269

### CODE REVIEW

File: src/primitive-field/primitive-field.component.ts [hunk 1]
```diff
@@ -114,6 +114,7 @@ export class PrimitiveFieldComponent extends AbstractFieldComponent implements O
         this.schema.onValueChange(this.path, this.value, this.jsonStoreService, this.keysStoreService)
```
Comment (drjova) [inline src/primitive-field/primitive-field.component.ts:114]: I would add it here, only if there is `onValueChange`

**Comment by michamos** [comment]
just to make sure: is this automatically stripping whitespace at the beginning and end?

**Comment by MJedr** [comment]
no, it would be done in editor in inspirehep repo

**Comment by michamos** [comment]
ok, just making sure there's no misunderstanding :)

**Review by drjova — APPROVED** [review]

**Pull request merged**